docs/en/remote-flow.rst

```diff
@@ -397,7 +397,7 @@ Below a non-normative example of the request:
   response=eyJhbGciOiJFUzI1NiIs...9t2LQ
   
 
-Below is a non-normative example of the decrypted JSON ``response`` content:
+Below is a non-normative example of the decrypted payload of the JWT contained in the ``response``, before base64url encoding and signing:
```

Comment:
```suggestion
Below is a non-normative example of the decrypted payload of the JWT contained in the ``response``, before base64url encoding:
```

it is not clear how the RP may obtain the wallet instance public keys if it will not ask the WIA

Reply:
In our current approach the RP may ask for the Wallet Attestation in the scope parameter within the request object. We may change approach if you want but I suggest to open an issue for this.